homeassistant/components/remote/harmony.py

```diff
@@ -142,23 +151,30 @@ def __init__(self, name, host, port, activity, out_path, delay_secs):
         self._state = None
         self._current_activity = None
         self._default_activity = activity
-        self._client = pyharmony.get_client(host, port, self.new_activity)
+        # self._client = pyharmony.get_client(host, port, self.new_activity)
```

Comment:
I just commented that out for now. Plan is to update pyharmony to be able to use callback again so that HASS does not need to poll every 5 seconds for a change. Next update should be able to include this. :-)